db/views/analytics_client_geolocations_v01.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+    id,
+    client_id,
+    source_type,
+    source_id,
+    located_on,
+    lat,
+    lon,
+    -- collected_by,
```

Comment:
For HMIS-collected Locations `collected_by` gets filled as the Project Name. However they are tied to an `Enrollment` via `source_id/`source_type`, so the project name is redundant. So I agree with leaving it out. (It might be useful for some warehouse installations that import(ed) location data, though - I'm unsure. But I don't think we should prioritize that for the analytics view).